GoogleForCreators/web-stories-wp
# Update plugin activation message styling

## Summary

Updates the plugin activation notices as defined in [Figma](https://www.figma.com/file/EHkkPjNMRANg7RD6dKJ3QZ/Stable-Marketing?node-id=558%3A855), see https://github.com/google/web-stories-wp/issues/3906#issuecomment-689356994

## Relevant Technical Choices

* Added `srcset` attribute to images
* Adapted Figma design to make it responsive.  
  ⚠️ This is important, as the Figma design is > 1880px wide, but this notice needs to work on smaller screens too. It's quite possible and expected that the resulting layout will not look exactly as on Figma all the time.

## To-do



## User-facing changes

Updated message upon plugin activation.

Some screenshots:
<img width="703" alt="Screenshot 2020-09-21 at 11 18 17" src="https://user-images.githubusercontent.com/841956/93751187-f93bd600-fbfc-11ea-8342-9c41b89b4ade.png">
<img width="861" alt="Screenshot 2020-09-21 at 11 17 05" src="https://user-images.githubusercontent.com/841956/93751191-f9d46c80-fbfc-11ea-9486-624caada2ed8.png">
<img width="263" alt="Screenshot 2020-09-21 at 11 16 58" src="https://user-images.githubusercontent.com/841956/93751193-fb059980-fbfc-11ea-8213-fef9fa729b00.png">

<img width="813" alt="Screenshot 2020-09-21 at 11 18 42" src="https://user-images.githubusercontent.com/841956/93751176-f640e580-fbfc-11ea-8909-da816c1cd331.png">


## Testing Instructions

1. Check out this branch
1. Run `npm run storybook`
1. Select "Plugin Activation" in sidebar
1. Run `npm run build`
1. Deactivate plugin
1. Activate plugin

_or_

1. Select this PR on the QA site
1. Deactivate plugin
1. Activate plugin

---



Fixes #3906
